opt/pyproject.toml

```diff
@@ -51,6 +51,7 @@ requests = "^2.22.0"
 fastapi = "^0.54.1"
 hypercorn = "^0.9.5"
 methodtools = "^0.1.2"
+imohash = "^1.0.4"
```

Comment:
I think this is a _very_ unwise change. Why not use `hashlib`? It's builtin

Reply:
Also from `imohash` docs:
# Misuses
Because imohash only reads a small portion of a file's data, it is not suitable for:
* file verification or integrity monitoring